core/init.go

```diff
@@ -0,0 +1,44 @@
+package core
+
+import (
+	"context"
+	"log"
+
+	"github.com/tokopedia/code-review-workshop/database"
+	redislib "github.com/tokopedia/code-review-workshop/redis"
+	"github.com/tokopedia/sqlt"
+)
+
+type PreparedStatements struct {
+	GetNilaiSiswaByID *sqlt.Stmtx
+}
+
+var (
+	db    *sqlt.DB
+	stmt  PreparedStatements
+	redis *redislib.RedisStore // redis module
```

Comment:
```suggestion
	redis *redislib.RedisStore
```
remove unnecessary comment